### PR TITLE
changed machines configuration, added new cluster

### PR DESCRIPTION
**Pull request by SergeyPod**

### CODE REVIEW

**Comment by hsinfang** [comment]
May you please squash commits of the same logical change together? Force-pushing to the ticket branch is fine. Details on the Dev Guide: https://developer.lsst.io/work/flow.html

**Review by hsinfang — CHANGES_REQUESTED** [review]
Please remember to rebase onto the new master before merging

File: modules/gke/variables.tf [hunk 1]
```diff
@@ -198,3 +198,24 @@ variable "node_pools_labels" {
...
+
+variable "cluster_autoscaling" {
+   type = object({
+     enabled = bool
+     autoscaling_profile = string
+     min_cpu_cores = number
+     max_cpu_cores = number
+     min_memory_gb = number
+     max_memory_gb = number
+   })
```
Comment (hsinfang) [inline modules/gke/variables.tf:211]: May you please add a brief description for this new variable?

File: modules/gke/variables.tf [hunk 2]
```diff
@@ -198,3 +198,24 @@ variable "node_pools_labels" {
     }
   }
 }
```
[the remark this answers is not in the document]
Comment (noahdungey) [reply modules/gke/variables.tf:200]: Changes to modules/gke/main.tf are not required as it sources terraform-google-modules/kubernetes-engine which has cluster_autoscaling defined as an Input. 
https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/latest/submodules/beta-private-cluster-update-variant?tab=inputs

**Review by hsinfang — APPROVED** [review]

**Pull request merged**